### PR TITLE
Add tests for TimeSpanMillisecondsGraphType/TimeSpanSecondsGraphType

### LINKED ISSUE #2355: Add negative tests for `ParseValue` and `ParseLiteral` for `TimeSpanMillisecondsGraphType` and `TimeSpanSecondsGraphType`
I don's see any negative tests for `ParseValue` and `ParseLiteral` for `TimeSpanMillisecondsGraphType` and `TimeSpanSecondsGraphType`. Either add them or let's add all lost test cases in separate PR with the help of codecov coverage colored map.

_Originally posted by @sungam3r in https://github.com/graphql-dotnet/graphql-dotnet/pull/2341#discussion_r588126925_

### PR DESCRIPTION
**Pull request by sungam3r**
fixes #2355

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -20,10 +20,6 @@
     <NoWarn>$(NoWarn);1591</NoWarn> <!--TODO: temporary solution-->
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Include="..\..\assets\logo.64x64.png" Pack="true" PackagePath="\" Condition="'$(IsPackable)' == 'true'" />
```
Comment (sungam3r) [inline src/Directory.Build.props:24]: Ooops, duplicate.

File: src/GraphQL.Tests/Types/TimeSpanMillisecondsGraphTypeTests.cs [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
+        [ClassData(typeof(TimeSpanMillisecondsGraphTypeTestsData))]
+        public void serialize_numerics(object value)
         {
             CultureTestHelper.UseCultures(() =>
             {
-                int input = 1;
-                var actual = _type.Serialize(input);
-                actual.ShouldBe(input);
+                var actual = _type.Serialize(value);
+                actual.ShouldBeOfType<long>().ShouldBe(value is BigInteger b ? (long)b : Convert.ToInt64(value));
```
Comment (sungam3r) [inline src/GraphQL.Tests/Types/TimeSpanMillisecondsGraphTypeTests.cs:49]: `BigInteger` does not implement `IConvertible`, `Convert.ToInt64` throws.

**Review by Shane32 — APPROVED** [review]

**Comment by Shane32** [comment]
As these are just tests, feel free to merge when you're satisfied with them.  I'll review further if there's any changes to the codebase.

**Comment by sungam3r** [comment]
изображение
Mission accomplished.

**Pull request merged**